ygen/gogen.go

```diff
@@ -1280,10 +1280,10 @@ func writeGoStruct(targetStruct *Directory, goStructElements map[string]*Directo
 			scalarField := true
 			fType := mtype.NativeType
 			schemapath := util.SchemaTreePathNoModule(field)
-			zeroValue := mtype.zeroValue
-			defaultValue := goLeafDefault(field, mtype)
+			ZeroValue := mtype.ZeroValue
```

Comment:
Same comment here as in `genstate.go`, we don't need to change the naming here within the function.